docs/en/sql-reference/functions/url-functions.md

```diff
@@ -464,5 +464,39 @@ Removes the query string and fragment identifier. The question mark and number s
 
 ### cutURLParameter(URL, name)
 
-Removes the ‘name’ URL parameter, if present. This function works under the assumption that the parameter name is encoded in the URL exactly the same way as in the passed argument.
+Removes the `name` URL parameter, if present. This function works under the assumption that the parameter name is encoded in the URL exactly the same way as in the passed argument.
```

Comment:
Agree. The original text needs some kind of "refactoring" / re-phrasing. 

But your proposal also does not clarify the text. Because usually when we talk about URL we say: URL **encoding** or **decoding**, not escaping unescaping (furthermore by spell check highlights "unescape").  

Second: parameter names may contain spaces inside. I know some developers use such parameter names. For example:

Parameter name Client ID is URL-encoded as Client%20ID

Maybe we should use this example in the documentation.

So it's hard to imagine parameter having = or + inside its name. (for parameter value it's OK, it's possible).

I am trying to find out now Proposal No 3 and Proposal No 4. Maybe you may help with it.

Reply:
I am fine with the "encoding/decoding" terminology instead of "escape/unescape" and I also like the Client ID example better. [Here](https://en.wikipedia.org/wiki/Percent-encoding#Reserved_characters) are the reserved characters. For the example, what about `Client ID=37740:12` (plaintext) and `Client%20ID=37740%3A12` (encoded).

Reply:
How about Proposal No 3:

This function does not encode or decode characters in parameter names, e.g. `Client ID` and `Client%20ID` are treated as different parameter names.

In your example above `Client%20ID=37740%3A12`:
`Client%20ID` is parameter's name
`37740%3A12` is parameter's value.
As the second argument for the function we only pass parameter's name `Client%20ID`. That's why we should not mention parameter's value `37740%3A12`  in the documentation. What do you think?

Reply:
Done